docs/current_docs/configuration/cloud.mdx

```diff
@@ -19,7 +19,7 @@ Dagger Cloud requires Dagger v0.11 or higher.
 
 ### Create a Dagger Cloud organization
 
-1. Sign up for Dagger Cloud by selecting a plan on the [Dagger website](https://www.dagger.io/pricing). Dagger Cloud includes plans for both individuals and teams. Click **Continue with GitHub** to log in with your GitHub account.
+1. Sign up for Dagger Cloud by selecting a plan on the [Dagger website](https://www.dagger.io/pricing). Dagger Cloud includes plans for both individuals and teams. Click `Continue with GitHub` to log in with your GitHub account.
```

Comment:
That URl no longer works, it should be /cloud

Reply:
Done, thanks!